### ISSUE TITLE
Error while executing keybase commands / cannot sign my pgp key via keybase commandline

### ISSUE THREAD

**Issue by random-access** [keybase/keybase-issues#1775]
I uploaded my PGP public key block at my profile page, then tried to sign it via commandline with keybase.

First time I executed it, "keybase login" worked. But an exception was thrown while executing "keybase push", same with "keybase push --update"

Now I always get the following stacktrace, already while executing keybase login:

```
warn: You have not proven ownership of your key. Run `keybase push --update`.
TypeError: Object #<ParsedKeys> has no method 'get_key_manager'
    at /usr/local/lib/node_modules/keybase/lib/user.js:1570:44
    at Deferrals.exports.Deferrals.Deferrals._call (/usr/local/lib/node_modules/keybase/node_modules/iced-runtime/lib/runtime.js:86:16)
    at /usr/local/lib/node_modules/keybase/node_modules/iced-runtime/lib/runtime.js:98:26
    at exports.trampoline.trampoline (/usr/local/lib/node_modules/keybase/node_modules/iced-runtime/lib/runtime.js:64:14)
    at Deferrals.exports.Deferrals.Deferrals._fulfill (/usr/local/lib/node_modules/keybase/node_modules/iced-runtime/lib/runtime.js:96:16)
    at _this.sibkeys (/usr/local/lib/node_modules/keybase/lib/user.js:1563:28)
    at User.exports.User.User._verify (/usr/local/lib/node_modules/keybase/lib/user.js:1565:15)
    at iced.Deferrals.parent (/usr/local/lib/node_modules/keybase/lib/user.js:791:32)
    at /usr/local/lib/node_modules/keybase/lib/user.js:798:27
    at Deferrals.exports.Deferrals.Deferrals._call (/usr/local/lib/node_modules/keybase/node_modules/iced-runtime/lib/runtime.js:86:16)
```
"keybase push --update" leads to exactly the same stacktrace.

```
monika@monika-tuxedo:/usr/local$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 15.04
Release:        15.04
Codename:       vivid
monika@monika-tuxedo:/usr/local$ uname -a
Linux monika-tuxedo 3.19.0-28-generic #30-Ubuntu SMP Mon Aug 31 15:52:51 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux
monika@monika-tuxedo:/usr/local$ nodejs --version
v0.10.25
monika@monika-tuxedo:/usr/local$ keybase --version
0.8.19

```

**Comment by maxtaco** [comment]
Looks like you were able to work around this problem?

**Comment by random-access** [comment]
Yes :) - I used the third method to verify my key (with curl and gpg), afterwards all keybase commands worked. I think I just might have messed it up with providing my key via the website and trying to verify it via commandline from my client...

**Comment by maxtaco** [comment]
Cool, glad to here it finally worked out.  We have code to handle that
corner case, but it broke with our most recent release.  We have a PR open
for it now. Thanks so much for your feedback!

**Comment by maxtaco** [comment]
Should be fixed in v0.8.20.  Let me know if not.
